plugins/feeds/public/blocklistde_all.py

```diff
@@ -1,31 +1,37 @@
 import logging
 from datetime import timedelta, datetime
-from core.errors import ObservableValidationError
-from core.feed import Feed
-from core.observables import Ip
+from core.schemas import observable
+from core.schemas import task
+from core import taskmanager
 
 
-class BlocklistdeAll(Feed):
-    default_values = {
+
+class BlocklistdeAll(task.FeedTask):
+    SOURCE = "https://lists.blocklist.de/lists/all.txt"
+    _defaults = {
         "frequency": timedelta(hours=1),
         "name": "BlocklistdeAll",
-        "source": "https://lists.blocklist.de/lists/all.txt",
         "description": "All IP addresses that have attacked one of our customers/servers in the last 48 hours. It's not recommended to use this feed due to the lesser amount of contextual information, it's better to use each blocklist.de feed separately.",
     }
 
-    def update(self):
-        for line in self.update_lines():
-            self.analyze(line)
+    def run(self):
+        response = self._make_request(self.SOURCE, verify=True)
+        if response:
+            data = response.text
+            for item in data.split("\n"):
+                self.analyze(item)
 
     def analyze(self, item):
         ip = item.strip()
 
-        context = {"source": self.name, "date_added": datetime.utcnow()}
+       
 
         try:
-            obs = Ip.get_or_create(value=ip)
-            obs.add_context(context, dedup_list=["date_added"])
-            obs.add_source(self.name)
-            obs.tag("blocklistde")
-        except ObservableValidationError as e:
+            obs = observable.Observable.find(value=ip)
+            if not obs:
+                obs = observable.Observable(value=ip, type="ip").save()
+            obs.tag(["blocklist"])
+
+        except Exception as e:
```

Comment:
This will make the feed fail silently, and the users will never know. Unless you expect this exception to fire a lot while parsing the feed (e.g. if IP values are often malformatted), I would just let the exception be raised (remove try / except) so it's logged and reported in the UI.

Reply:
Yep I'll fix it